### ISSUE TITLE
This module moves the background to top and left

### ISSUE THREAD

**Issue by naseerfaheem** [bibaldo/MMM-Pregnancy#5]
After activating this module, it shifts the screen to top and left sides. I think the issue is in the w3.css file. Can we have a fix for that? thanks,

**Comment by TheDuffman85** [comment]
@naseerfaheem You are right, this is because of the w2.css. I've fixed that in my [fork](https://github.com/TheDuffman85/MMM-Pregnancy).
